dataflow/src/main/java/org/checkerframework/dataflow/livevariable/package-info.java

```diff
@@ -0,0 +1,9 @@
+/**
+ * Classes using for live variable analysis. Live variable analysis is a backward analysis to
+ * calculate the variables that are live at each point in the program. To run live variable
+ * analysis, see {@link org.checkerframework.dataflow.cfg.playground.LiveVariablePlayground}.
```

Comment:
Thanks for the comment! I would expect people to run the `LiveVariablePlayground`. It now has been mentioned in the manual.
I am not sure what to do with this `package-info.java`, I just wanted to point the users to `LiveVariablePlayground`. Do you mean that I need to copy and paste `LiveVariablePlayground`'s code here?

Reply:
I see the livevariable backwards analysis like the constantpropagation forward analyses: demo analyses for the dataflow framework that are only used in their respective playgrounds. Either one could be useful to users.
If we decide to move the livevariable implementation we should also move the constantpropagation and the playgrounds to a test directory.

I like the suggestion of adding a section to the dataflow manual, like for constantpropagation: https://github.com/typetools/checker-framework/blob/master/dataflow/manual/content.tex#L1699
As this PR contains the tests for the backwards analysis implementation, it might be good to merge it and work on the manual in a separate PR. But it's also not critical to get this PR into the release, so we can work on the manual section in this PR.

Reply:
Oh, this PR already adds it to the dataflow manual. So that's not a blocker.